flowmachine/flowmachine/core/query.py

```diff
@@ -70,7 +77,7 @@ def __init__(self, cache=True):
         obj = Query._QueryPool.get(self.query_id)
         if obj is None:
             try:
-                self.connection
+                get_db()
             except AttributeError:
```

Comment:
This should definitely be removed cos you'll never hit the exception - `get_db` will raise it by itself.